### DISCUSSION TITLE
Assets not found

### DISCUSSION THREAD

**Discussion by jhavenz** [filamentphp/filament#457] [General]
Hi guys, 

First off, thank you for the great project. Really have enjoyed working with it so far.

I'm not sure if this falls under a bug or if I'm doing something wrong but I have Filament working locally, then when I deploy
(I use a site called cleavr.io ... which creates a 'current' folder that the document root resides in '/home/myapp/current'), I am having issues with a 404 error as far as the Filament assets go.
The strange part is... when I look in the console, I see the 404 from the assets not being found. But when I copy and paste the url into a new tab in the browser, the assets pop right up.
Hopefully im making sense and that the GIF ive attached adds some extra context.

![filament-404s](https://user-images.githubusercontent.com/53377590/121851791-6a4b7e00-cca3-11eb-8d40-c8c4c32a9d63.gif)

**Comment by loranger** [comment]
Interesting, I have the exact same bug.

I thought it was because of the server configuration. 
Once deployed the page refuse to load the assets.
The files (livewire.js and filament.js) are considered unreachable, whereas loading the requested url do respond with their content.

It results with a `login:239 Uncaught ReferenceError: Livewire is not defined` error.

I've create [an issue](https://github.com/laravel-filament/filament/issues/458). 
I hope this one won't be closed and convert as a discussion…

**Comment by Iamkhaledkhan** [comment]
did anyone fixed it? please let me know, i have the same issue.

**Comment by wolfcoder** [comment]
3 years from the initial issue. I came here with the same problem, and don't get answer

**Comment by alkhachatryan** [comment]
I know it may be useless for you, but usefull for a people now with the same issue. Did you set the ASSET_URL in env file? And also you need to tell your virtual host to try to read the static files from the path, if not exist - then go to dynamic router of laravel
